plugins/funder.c

```diff
@@ -219,16 +220,20 @@ json_openchannel2_sign_call(struct command *cmd,
 	/* Use input markers to identify which inputs
 	 * are ours, only sign those */
 	json_array_start(req->js, "signonly");
+	count = 0;
 	for (size_t i = 0; i < psbt->num_inputs; i++) {
-		if (psbt_input_is_ours(&psbt->inputs[i]))
+		if (psbt_input_is_ours(&psbt->inputs[i])) {
 			json_add_num(req->js, NULL, i);
+			count++;
+		}
 	}
 	json_array_end(req->js);
 
 	plugin_log(cmd->plugin, LOG_DBG,
-		   "calling `signpsbt` for channel %s",
+		   "calling `signpsbt` for channel %s for %zu input%s",
```

Comment:
Maybe there is missing space in `input%s`?

Reply:
The last `%s` is for gramamtical reasons. It'll print "1 input" or "2 inputs".

Reply:
My big mistake, sorry I only take into consideration the line and not the meaning! (bad job vincent :) ).